docs/timelines/running-and-editing-timelines.md

```diff
@@ -0,0 +1,138 @@
+---
+id: running-and-editing-timelines
+title: Running and Editing Timelines
+sidebar_label: Running and Editing Timelines
+---
+
+
+Running a timeline to timecode
+------------------
+
+The HTP (intensity) of fixtures in the timeline is controlled by the fader level of the handle. The console will automatically fire the timeline playback to full when timecode is received within the range of the timeline settings (you can turn this off using the option [Autoload in Range](../timelines/timeline-options.md#timecode-tab)).
+
+1.  Ensure the **timecode link** button (bottom left of the timeline view) is set to **Linked**.
+
+2.  Start the timecode source. (You will need to push up the timeline fader for internal timecode or if you have disabled the autoload option described above).
```

Comment:
Do we not auto-activate if the timecode source is internal? That sounds like it might be a bug?

Reply:
As far as I can see the internal timecode does not activate by default. You can set the Fader Raised Action option to Play to make this happen but it isn't the default setting. Tell me if I am missing somethinig.

Reply:
When we say activate we're talking about not needing to push up the fader. i.e. when you start the timecode source the fader should automatically go to full. So this part is not needed: `(You will need to push up the timeline fader for internal timecode or if you have disabled the autoload option described above).`

Reply:
OK, should I leave in "You will need to push up the timeline fader if you have disabled the auto-activate option described above"

Reply:
Yep, that part is correct 👍